x-pack/plugins/endpoint/server/plugin.ts

```diff
@@ -11,6 +11,7 @@ import { createConfig$, EndpointConfigType } from './config';
 import { registerEndpointRoutes } from './routes/endpoints';
 import { EndpointAppContext } from './types';
 import { registerAlertRoutes } from './routes/alerts';
+import { registerResolverRoutes } from './routes/resolver';
```

Comment:
Can you group all the `routes` imports together here? That was probably my fault, but `types` should probably come before all the `routes` imports.